openshift/templates/dancer-mysql.json

```diff
@@ -330,6 +334,11 @@
       "value": "openshift/mysql-55-centos7"      
     },
     {
+      "name": "PERL_APACHE2_RELOAD",
+      "description": "Set this to anything other than \"false\" or \"0\" to enable automatic reloading of modified Perl modules",
```

Comment:
again, this is wrong... should be "Set this to \"true\" to enable automatic reloading of source code"